package.json

```diff
@@ -256,9 +256,11 @@
     "@metamask/network-controller@npm:^19.0.0": "patch:@metamask/network-controller@npm%3A19.0.0#~/.yarn/patches/@metamask-network-controller-npm-19.0.0-a5e0d1fe14.patch",
     "@metamask/nonce-tracker@npm:^5.0.0": "patch:@metamask/nonce-tracker@npm%3A5.0.0#~/.yarn/patches/@metamask-nonce-tracker-npm-5.0.0-d81478218e.patch",
     "@metamask/gas-fee-controller@npm:^15.1.1": "patch:@metamask/gas-fee-controller@npm%3A15.1.2#~/.yarn/patches/@metamask-gas-fee-controller-npm-15.1.2-db4d2976aa.patch",
-    "@metamask/snaps-utils@npm:^7.6.0": "patch:@metamask/snaps-utils@npm%3A7.6.0#~/.yarn/patches/@metamask-snaps-utils-npm-7.6.0-5569b09766.patch",
+    "@metamask/snaps-utils@npm:^7.7.0": "patch:@metamask/snaps-utils@npm%3A7.7.0#~/.yarn/patches/@metamask-snaps-utils-npm-7.7.0-2cc1f044af.patch",
     "@metamask/snaps-utils@npm:^7.4.0": "patch:@metamask/snaps-utils@npm%3A7.6.0#~/.yarn/patches/@metamask-snaps-utils-npm-7.6.0-5569b09766.patch",
```

Comment:
Hmm, these should be using 7.7.0 as well. We don't need both minor versions.

Reply:
Resolved in https://github.com/MetaMask/metamask-extension/pull/26519/commits/95af466efa8b0b129e1f95ca767ec32594c6bcf5

Reply:
@MetaMask/snaps-devs could you double-check that this makes sense? I don't think there's a reason we'd want v7.6.0 and v7.7.0 both present, but would appreciate confirmation of that.

Reply:
Seems fine to me, but will let @Mrtenz respond as well as he wrote the patch!

Reply:
Looks good! Not sure why it was duplicated in the first place.